Makefile

```diff
@@ -100,32 +104,47 @@ $(subst deb,%,$(DEBS)): $(DEB_TOP)/.patched
 	  $(DEB_BUILD)/debian/copyright
 	sed -i 's#url://example.com#$(PROJECT_URL)#' \
 	  $(DEB_BUILD)/debian/copyright
-	echo "On Debian Systems, the full text of the GNU General Public" \
+	echo "License: GPL-2 and GPL-3 notices" \
```

Comment:
Debmake tries to find all copyright notices in the source code and puts them in a single copyright file.  Per the packaging standard if it finds references to GPL 2 or GPL 3 in any of the license packages, it expects that these lines should also show up in that license file.  They do not for argobots.

I have changed to a different building since debmake is apparently deprecated.